src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -50,6 +50,40 @@
 import teammates.storage.api.StudentsDb;
 
 public class FeedbackSessionsLogic {
+    
+    private static final String ASSUMPTION_FAIL_DELETE_INSTRUCTOR = "Fail to delete instructor respondant for ";
+    private static final String ASSUMPTION_FAIL_RESPONSE_ORIGIN = "Client did not indicate the origin of the response";
+    private static final String ASSUMPTION_FAIL_RESPONSES_ORIGIN = "Client did not indicate the origin of the responses";
+    private static final String ERROR_NUMBER_OF_RESPONSES_EXCEEDS_RANGE = "Number of responses exceeds the limited range";
+    private static final String ERROR_SENDING_EMAILS = "Error while sending emails :";
+    private static final String ERROR_SESSION_UNPUBLISH_ALREADY = "Session is already unpublished.";
+    private static final String ERROR_SESSION_UNPUBLISH_NON_EXISTENT = "Trying to unpublish a non-existent session.";
+    private static final String ERROR_SESSION_UNPUBLISH_PRIVATE = "Private session can't be unpublished.";
+    private static final String ERROR_SESSION_PUBLISH_ALREADY = "Session is already published.";
+    private static final String ERROR_SESSION_PUBLISH_NON_EXISTENT = "Trying to publish a non-existent session.";
+    private static final String ERROR_SESSION_PUBLISH_PRIVATE = "Private session can't be published.";
+    private static final String ERROR_GET_NON_EXISTENT_SESSION_COURSE = "Trying to get feedback sessions for a course that does not exist.";
+    private static final String ERROR_SESSION_DOES_NOT_EXIST_STRING_FORMAT = "Trying to %s a feedback session %sthat does not exist.";
+    private static final String ERROR_NON_EXISTENT_SESSION_STRING_FORMAT = "Trying to %s non-existent feedback session%s";
+    private static final String FEEDBACK_SESSION_DOES_NOT_EXIST_CHECK = "check";
```

Comment:
The names for these constants could be changed to `ACTION_<action string>` to show that the type of the string it contains is an action that was performed. i.e. `ACTION_CHECK`, `ACTION_UPDATE`, `ACTION_GET`, `ACTION_REMIND`, `ACTION_VIEW`